app/views/assessments/_downloadSubmission.html.erb

```diff
@@ -1,15 +1,17 @@
 <div class="history-download-pane" style="margin-right: 30px;">
   <% if can_always_download or not (sub.assessment.exam or sub.assessment.course.exam_in_progress) then %>
     <% mime_type = sub.detected_mime_type %>
-    <%= link_to download_course_assessment_submission_path(@course, @assessment, sub),
-                data: {toggle: "tooltip", placement:"top"}, title: "Download Submission", style:"white-space: nowrap" do %>
-      <span style="margin-right:3px;"><i class="material-icons">file_download</i></span>
-      Download Submission
-    <% end %>
     <% if mime_type =~ /text\/.*/ then %>
```

Comment:
what if mime_type is nil? 
I checked and it seems that if it's `nil =~ /text/\.*/` it will return false so I think it should be okay?

Reply:
Tested, it does return false so it's fine